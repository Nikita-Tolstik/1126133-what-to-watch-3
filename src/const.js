export const MarkFilm = {
  BAD: `Bad`,
  NORMAL: `Normal`,
  GOOD: `Good`,
  VERY_GOOD: `Very good`,
  AWESOME: `Awesome`,
};

export const AppRoute = {
  ROOT: `/`,
  LOGIN: `/login`,
  MY_LIST: `/mylist`,
  FILMS: `/films`,
  REVIEW: `/review`,
  PLAYER: `/player`,
};

export const Error = {
  BAD_REQUEST: 400,
  UNAUTHORIZED: 401,
  SERVER: 500,
};
