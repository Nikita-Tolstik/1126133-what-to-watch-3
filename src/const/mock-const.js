export const MockSettings = {
  TITLE: `The Grand Budapest Hotel`,
  GENRE: `Drama`,
  YEAR: 2014
};

export const mockTitles = [

  `Fantastic Beasts: The Crimes of Grindelwald`,
  `Bohemian Rhapsody`,
  `Macbeth`,
  `Aviator`,
  `We need to talk about Kevin`,
  `What We Do in the Shadows`,
  `Revenant`,
  `Johnny English`
];
