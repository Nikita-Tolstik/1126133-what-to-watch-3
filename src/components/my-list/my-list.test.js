import React from 'react';
import renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import MyList from './my-list.jsx';
import history from '../../history.js';
import configureStore from "redux-mock-store";
import {Provider} from 'react-redux';
import NameSpace from '../../reducer/name-space.js';
import {AppRoute} from '../../const.js';

const mockStore = configureStore([]);

const mockFilms = [
  {
    id: 1,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 2,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 3,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 4,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 5,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 6,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 7,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 8,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 9,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 10,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 11,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
  {
    id: 12,
    title: `Fantastic Beasts: The Crimes of Grindelwald2`,
    posterImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundImage: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    backgroundColor: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Fantastic Beasts: The Crimes of Grindelwald`,
    rating: 5,
    scoresCount: 7,
    director: `Wes Andreson`,
    stars: [`Bill Murray`, `Edward Norton`, `Jude Law`],
    runTime: 55,
    genre: `Action`,
    released: 44,
    isFavorite: true,
    videoLink: `https://upload.wikimedia.org/`,
    videoPreview: `https://upload.wikimedia.org/`,
  },
];

const children = <div className="children-component" />;

it(`Render MyList with films`, () => {

  history.push(AppRoute.MY_LIST);

  const store = mockStore({
    [NameSpace.DATA]: {
      favoriteFilms: mockFilms,
      isError: false,
    },
    [NameSpace.USER]: {
      authorizationStatus: `AUTH`,
      userInfo: `xxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
    }
  });

  const tree = renderer.create(
      <Router history={history}>
        <Provider store={store}>
          <MyList
            onCardFilmClick={() => {}}
          >
            {children}
          </MyList>
        </Provider>
      </Router>
  );

  expect(tree).toMatchSnapshot();

});


it(`Render MyList without films`, () => {

  history.push(AppRoute.MY_LIST);

  const store = mockStore({
    [NameSpace.DATA]: {
      favoriteFilms: [],
      isError: true,
    },
    [NameSpace.USER]: {
      authorizationStatus: `AUTH`,
      userInfo: `xxxxxxxxxxxxxxxxxxxxxxxxxxxxx`,
    }
  });

  const tree = renderer.create(
      <Router history={history}>
        <Provider store={store}>
          <MyList
            onCardFilmClick={() => {}}
          >
            {children}
          </MyList>
        </Provider>
      </Router>
  );

  expect(tree).toMatchSnapshot();

});
