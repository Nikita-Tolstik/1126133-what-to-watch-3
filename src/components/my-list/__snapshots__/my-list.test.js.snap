// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MyList with films 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link "
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    >
      My list
    </h1>
    <div
      className="user-block"
    >
      <a
        href="/mylist"
        onClick={[Function]}
      >
        <div
          className="user-block__avatar"
        >
          <img
            alt="User avatar"
            height="63"
            src="https://htmlacademy-react-3.appspot.comxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
            width="63"
          />
        </div>
      </a>
    </div>
  </header>
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <div
      className="catalog__movies-list"
    >
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/1"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/1"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/2"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/2"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/3"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/3"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/4"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/4"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/5"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/5"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/6"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/6"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/7"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/7"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/8"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/8"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/9"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/9"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/10"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/10"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/11"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/11"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <a
          href="/films/12"
          onClick={[Function]}
        >
          <div
            className="small-movie-card__image"
            onClick={[Function]}
          >
            <video
              height="175"
              muted={true}
              poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
              width="280"
            />
          </div>
        </a>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="/films/12"
            onClick={[Function]}
          >
            Fantastic Beasts: The Crimes of Grindelwald2
          </a>
        </h3>
      </article>
    </div>
  </section>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        © 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
  <div
    className="children-component"
  />
</div>
`;

exports[`Render MyList without films 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link "
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    >
      My list
    </h1>
    <div
      className="user-block"
    >
      <a
        href="/mylist"
        onClick={[Function]}
      >
        <div
          className="user-block__avatar"
        >
          <img
            alt="User avatar"
            height="63"
            src="https://htmlacademy-react-3.appspot.comxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
            width="63"
          />
        </div>
      </a>
    </div>
  </header>
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <div
      className="catalog__movies-list"
    />
  </section>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        © 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
  <div
    className="children-component"
  />
</div>
`;
